apps/src/templates/GameButtons.jsx

```diff
@@ -16,16 +15,6 @@ const styles = {
     // to gameButtons. We want to get rid of that
     marginBottom: -18
   },
-  runImage: {
```

Comment:
Don't see re-addition elsewhere, assuming for non-minecraft apps this set of styles was never removed from the stylesheet?